### ISSUE TITLE
[JENKINS-19366] Configure tab freezes for projects

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-06-a2#6933]
<p>When clicking on the configuration tab of a project, the "LOADING" message stays there forever. The values shown on the configuration are wrong and it is impossible to edit the configuration of a project.</p>

<p>It is still possible to edit the configuration of a project manually by hacking up the xml file, but it is a very tedious process.</p>

**Attachments:**

- [jenkins-loading-js-problem.png](https://issues.jenkins.io/secure/attachment/24257/jenkins-loading-js-problem.png)

---
<details><summary><i>Originally reported by <a title="Carlos Duclos" href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=carlosduclos">carlosduclos</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-19366" target="_blank">Configure tab freezes for projects</a></i></summary>
<i><ul>
<li><b>reported by</b>: carlosduclos
<li><b>status</b>: Resolved
<li><b>priority</b>: Critical
<li><b>resolution</b>: Duplicate
<li><b>resolved</b>: 2014-05-20T21:01:00+00:00
<li><b>votes</b>: 2
<li><b>watchers</b>: 4
<li><b>imported</b>: 2025-11-06
</ul></i>
</details>

**Comment by jenkins-infra-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=tboven">tboven</a>:</i>
<p>I'm having the same issue.</p>

<p>During the testing I've done, I noticed that:</p>
<ul class="alternate" type="square">
	<li>the general configure-screen (of jenkins itself) works but the configure of the jobs is blocked by this issue</li>
	<li>the save-buttom becomes available (as soon as the screen isn't loading anymore but the overlay is still in front of the top of the page) but when I click it, I get "net::ERR_CONTENT_DECODING_FAILED"</li>
</ul>

**Comment by jenkins-infra-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=carlosduclos">carlosduclos</a>:</i>
<p>I upgraded to 1.530 and still the same. I installed a new Jenkins instance on a test machine and as long as I did not install any plug ins this issue was not present. The moment I installed plugins this issue started appearing.<br/>
At first I only had the plugins that come by default and the "Github pull request" plugin. With that configuration this issue did not appear.<br/>
This is the list of plugins I have installed (all of them via the "Manage Plugins" from "Manage Jenkins":</p>

<p>Ant Plugin 1.2			<br/>
Build Failure Analyzer 1.5.0			<br/>
Build Pipeline Plugin 1.4<br/>
build-metrics 1.0<br/>
conditional-buildstep 1.3.1<br/>
Credentials Plugin 1.7.6<br/>
Dashboard View 2.9.1<br/>
database 1.3<br/>
database-mysql 1.0<br/>
database-postgresql 1.0<br/>
External Monitor Job Type Plugin 1.2<br/>
Fingerprint Plugin 1.2<br/>
GitHub API Plugin 1.44<br/>
GitHub plugin 1.8<br/>
GitHub Pull Request Builder 1.9			<br/>
HTML Publisher plugin 1.2			<br/>
Hudson global-build-stats plugin 1.3<br/>
Javadoc Plugin 1.1<br/>
Jenkins Audit to Database Plugin 0.5<br/>
Jenkins Clone Workspace SCM Plug-in 0.6<br/>
Jenkins CVS Plug-in 2.9<br/>
Jenkins Doxygen Plug-in 0.14			<br/>
Jenkins Email Extension Plugin 2.32<br/>
Jenkins GIT client plugin 1.1.2<br/>
Jenkins GIT plugin 1.5.0<br/>
Jenkins jQuery plugin 1.7.2-1<br/>
Jenkins Mailer Plugin 1.5<br/>
Jenkins Multiple SCMs plugin 0.2<br/>
Jenkins Parameterized Trigger plugin 2.20<br/>
Jenkins SSH Slaves plugin 1.2<br/>
Jenkins Subversion Plug-in 1.50<br/>
Jenkins Translation Assistance plugin 1.10<br/>
LDAP Plugin 1.6<br/>
Maven Integration plugin 1.530			<br/>
PAM Authentication plugin 1.1<br/>
Publish Over SSH 1.10<br/>
Run Condition Plugin 0.10<br/>
SSH Credentials Plugin 1.4<br/>
Test stability history 1.0<br/>
Token Macro Plugin 1.8.1</p>

**Comment by jenkins-infra-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=carlosduclos">carlosduclos</a>:</i>
<p>I went through all the plugins, enabling and disabling them. After several tests I can say with some certainty that the Publish Over SSH plugin is causing the problem. As soon as I enable it, this problem comes back.</p>

**Comment by jenkins-infra-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=carlosduclos">carlosduclos</a>:</i>
<p>Given that this issue has happened with two different and unrelated plugins, it might be that the root cause is somewhere in the core.<br/>
My bet would be that the configuration freeze for projects is due to a plugin not providing what is expected and the loader not detecting that.</p>

**Comment by jenkins-infra-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=danielbeck">danielbeck</a>:</i>
<p>Given the versions mentioned in the comments, this is likely a duplicate of <a href="https://issues.jenkins.io/browse/JENKINS-19457" title="Buggy plugin implementations break job configuration too easily" class="issue-link" data-issue-key="JENKINS-19457"><del>JENKINS-19457</del></a> which was fixed in 1.531.</p>

**Comment by jenkins-infra-bot** [comment]
<i><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gmcdonald">gmcdonald</a>:</i>
<p>For your information, <a href="https://issues.jenkins.io/issues/?jql=project %3D JENKINS AND component %3D publish-over-ssh-plugin" class="external-link" rel="nofollow">all publish-over-ssh component type JENKINS issues</a> related to the <a href="https://plugins.jenkins.io/publish-over-ssh/" class="external-link" target="_blank" rel="nofollow noopener">Publish Over SSH plugin</a> have been transferred to Github: <a href="https://github.com/jenkinsci/publish-over-ssh-plugin/issues" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/publish-over-ssh-plugin/issues</a></p>

<p>Here is the direct link to this issue in Github: <a href="https://github.com/jenkinsci/publish-over-ssh-plugin/issues/186" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/publish-over-ssh-plugin/issues/186</a><br/>
And here is the link to a search for related issues: <a href="https://github.com/jenkinsci/publish-over-ssh-plugin/issues?q=%22JENKINS-19366%22" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/publish-over-ssh-plugin/issues?q=%22JENKINS-19366%22</a></p>

<p>(Note: this is an automated bulk comment)</p>

**Comment by jenkins-infra-bot** [comment]
<i>[Duplicates: <a href="https://github.com/timja-org/jenkins-gh-issues-poc-11-06-a2/issues?q=in%3Atitle%20JENKINS-19457">JENKINS-19457</a>]</i>

**Comment by jenkins-infra-bot** [comment]
<i>[Originally related to: <a href="https://github.com/timja-org/jenkins-gh-issues-poc-11-06-a2/issues?q=in%3Atitle%20JENKINS-19248">JENKINS-19248</a>]</i>
